doc-content/enterprise-only/getting-started/decision-table-penalty-columns-proc.adoc

```diff
@@ -0,0 +1,26 @@
+[id='penalty-columns-proc']
+= Inserting `Fine Amount` and `Points` columns
+
+The `Fine Amount` and `Points` columns contain the fines and points based on the corresponding `Violation Type` field values.
+
+.Prerequisite
+
+Inserted the `Violation Type` column.
+
+.Procedure
+. Click *Insert Column*, select *Set the value of a field* -> *Pattern*, and select *Violation[v]* from the *Pattern* pull-down menu.
+. Select *Field* and then *fineAmount* from the *Field* pull-down menu.
+. Select *Value options*, then select *Additional info* and enter `Fine Amount` in the *Header (description)* field.
+. Select the *Update engine with changes* option and click *Finish*.
++
+
+.Fine Amount header
+image::getting-started/fine-amount.png[]
+. Click *Insert Column*, select *Set the value of a field* -> *Pattern*, and select *Violation[v]* from the *Pattern* pull-down menu.
+. Select *Field* and then *points* from the *Field* pull-down menu.
+. Select *Value options*, then select *Additional info* and enter `Points` in the *Header (description)* field.
```

Comment:
....options, and then (or just and) select Additional info, and then (add comma after info)

Reply:
Done.